modular_skyrat/modules/goofsec/code/sol_fed.dm

```diff
@@ -323,30 +324,48 @@ GLOBAL_LIST_INIT(call911_do_and_do_not, list(
 
 	id_trim = /datum/id_trim/solgov
 
-/datum/antagonist/ert/request_911/fire
-	name = "Firefighter"
-	role = "Firefighter"
-	department = "Fire"
-	outfit = /datum/outfit/request_911/fire
-
-/datum/outfit/request_911/fire
-	name = "911 Response: Firefighter"
-	back = /obj/item/storage/backpack/duffelbag/cops
-	uniform = /obj/item/clothing/under/rank/engineering/atmospheric_technician
-	shoes = /obj/item/clothing/shoes/sneakers/yellow
+/datum/antagonist/ert/request_911/atmos
+	name = "Breach Control Technician"
+	role = "Breach Control Technician"
+	department = "Breach Control"
+	outfit = /datum/outfit/request_911/atmos
+
+/datum/outfit/request_911/atmos
+	name = "811 Response: Breach Control"
+	back = /obj/item/mod/control/pre_equipped/atmospheric
+	uniform = /obj/item/clothing/under/rank/engineering/engineer/hazard
+	shoes = /obj/item/clothing/shoes/workboots
 	ears = /obj/item/radio/headset/headset_eng
 	head = /obj/item/clothing/head/hardhat/red
-	suit = /obj/item/clothing/suit/fire/firefighter
```

Comment:
did you delete this by accident? they can't have anything in their suit store if they don't have a suit lol

Reply:
oh wait im dumb, MODsuits